src/forms/Form/Col.tsx

```diff
@@ -66,7 +66,22 @@ const Col = (props: ColPropsTypes) => {
 		imageUpload: FormFieldImageUpload,
 	}), []);
 
+	const doneTypingInterval = 500;
+	let typingTimer;
+
+	const sendValidateField = async (curr) => {
+		await dispatch(formActions.validateField({ name: curr.name }))
+
+		if (curr.pairedFields)
+			curr.pairedFields.forEach(async pairedField => {
+				await dispatch(
+					formActions.validateField({ name: pairedField })
+				);
+			});
+	};
+
 	const onChangeMap = useMemo(() => {
+		clearTimeout(typingTimer);

```

Comment:
The clearTimeout should go before the setTimeout (line 93) in order to actually clean it, because right now it waits until the "doneTypingInterval" value finishes but it gets executed the same amount of letters the input has. Also I think we can reduce the value from 500 to 300.